inputs/orchestrator_inner:6.json

```diff
@@ -59,12 +42,7 @@
   ],
```

Comment:
Let's make "resolve_composes" required by removing the ` "required": false` property.

Reply:
Fixed.